dbt/include/databricks/macros/adapters.sql

```diff
@@ -79,6 +85,10 @@
 {% macro databricks__create_view_as(relation, sql) -%}
   create or replace view {{ relation }}
   {{ comment_clause() }}
+  {%- set contract_config = config.get('contract') -%}
```

Comment:
what is the difference between tables and views? Which constraints are available in each one of them?

Reply:
In the dbt docs: Views offer limited support for column names and data types, but not constraints
So constraints are the same in tables and views.  For column names/data types that is handled by get_assert_columns_equivalent

Reply:
We should make sure that we do not issue ALTER TABLE statements for views, and that if the user defines constraints for views, we should log warnings saying that we do not support such thing.